internal/pkg/platform/model/branch.go

```diff
@@ -0,0 +1,164 @@
+// Code generated by ent, DO NOT EDIT.
+
```

Comment:
V tomto commite je vygenerovany kod, `11k` riadkov.

Obsahuje to DB migraciu - vytvorenie tabuliek, a rozne builders pre `create/update/query`.

Viac info v dokumentacii https://entgo.io/docs/getting-started